src/commands/apm_cmds/versions.js

```diff
@@ -10,12 +10,13 @@ exports.handler = async function ({ reporter, module, bump, cwd, network, apm: a
   const web3 = await ensureWeb3(network)
 
   apmOptions.ensRegistryAddress = apmOptions['ens-registry']
-
   const versions = await APM(web3, apmOptions).getAllVersions(module.appName)
   reporter.info(`${module.appName} has ${versions.length} published versions`)
   versions.map((version) => {
     if (version && version.content) {
       reporter.success(`${version.version}: ${version.contractAddress} ${version.content.provider}:${version.content.location}`)
+    } else if (version && version.error) {
+      reporter[version.reportType](`${version.version}: ${version.contractAddress} ${version.error}`)
```

Comment:
`reportType` is exposed in https://github.com/aragon/apm.js/pull/26 respectively.

Reply:
As commented in the apm.js issue, we can just report a warning in both cases, and break the current coupling of apm.js having to be aware of the supported reporters in the aragon-cli